pages/login.tsx

```diff
@@ -2,27 +2,43 @@ import Seo from 'components/Layout/Seo/Seo';
 import { getProtocol } from 'utils/urls';
 
 interface Props {
-  gssoUrl: string;
+  gssoUrl?: string;
```

Comment:
This should be mandatory

Reply:
i get a typescript problem down on line 46 when it's required:

```
(property) Props.gssoUrl: string
Type 'string | undefined' is not assignable to type 'string'.
  Type 'undefined' is not assignable to type 'string'.ts(2322)
login.tsx(5, 3): The expected type comes from property 'gssoUrl' which is declared here on type 'Props'
```

Reply:
just pushed a fixed for that, it's because we need to specify the env variable in order to be enabled in the typechecking https://github.com/LBHackney-IT/lbh-social-care/blob/lbh-frontend/next-env.d.ts#L7